spec/index.bs

```diff
@@ -4729,10 +4739,14 @@ Issue: convert "Valid Texture Copy Range" into an algorithm with parameters, sim
             If any of the following conditions are unsatisfied, generate a validation error and stop.
             <div class=validusage>
                 - |this|.{{GPUCommandEncoder/[[state]]}} is {{encoder state/open}}.
-                - [$validating GPUTextureCopyView$](|source|) returns `true`.
+                - [$validating GPUTextureCopyView$](|source|, |copySize|) returns `true`.
                 - |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[textureUsage]]}} contains
                     {{GPUTextureUsage/COPY_SRC}}.
                 - |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[sampleCount]]}} is 1.
+                - If |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}} is a depth-stencil format:
+                    - |source|.{{GPUTextureCopyView/aspect}} must refer to a single copyable aspect of
+                        |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}}.
+                        See [[#depth-formats|depth-formats]].
```

Comment:
As a followup, let's consider deduplicating this validation into a common place for B2T/T2B/writeTexture.